test/com/google/javascript/jscomp/Es6RewriteBlockScopedDeclarationTest.java

```diff
@@ -708,6 +708,46 @@ public void testDoWhileForOfCapturedLet() {
             "} while (false);"));
   }
 
+  public void testIssue1124() {
```

Comment:
I think there's a convention that "issue 1124" means 1124 on code.google.com. Let's add a comment with a link so it's unambiguous.